libsignal-service/src/models.rs

```diff
@@ -29,6 +28,6 @@ pub struct Message {
     pub message: String,
     pub attachments: Vec<Attachment<u8>>,
     pub group: Option<Group>,
-    pub timestamp: i64,
-    pub flags: i32,
+    pub timestamp: u64,
+    pub flags: u32,
```

Comment:
As stated elsewhere, I can't remember the story behind how the types came to be; there was something from Go and something ended up in the Diesel models, but I don't think either one of us consulted the protobufs until now.

Should make sense to use those as a starting point and if everything compiles cleanly from here on, that's just winning.

Reply:
Tested current Whisperfish against this and there is some work to do to get it to compile. But if this is what OWS is sending in, it's what should be, and the code adapt to it.

Reply:
Looking at the errors and thinking back, this would have been a side-effect of [Diesel and the db not using unsigned ints](https://users.rust-lang.org/t/diesel-use-of-derive-insertable-causes-a-compile-error/13838/5).

It's a good guess the Go version of Whisperfish does not care about this, because the database has no unsigned types yet the Go code never complained.

Doubt there's more to say here; I'll adapt Whisperfish's use of Diesel around this.